examples/angular/README.md

```diff
@@ -1,4 +1,4 @@
-![Angular Logo](../../packages/frameworks/logos/angular.svg)
+![Angular Logo](https://github.com/zeit/now/blob/master/packages/frameworks/logos/angular.svg)
```

Comment:
I'm surprised this works because the link goes to the preview page for editing, not to the raw image.

I would have expected https://raw.githubusercontent.com/zeit/now/master/packages/frameworks/logos/angular.svg